src/main/kotlin/no/nav/syfo/dialogmote/DialogmoteService.kt

```diff
@@ -126,7 +128,7 @@ class DialogmoteService(
             personIdent = personIdent,
             token = token,
             callId = callId,
-        )
+        ) && doesNotHaveAdressebeskyttelse(personIdent = personIdent, callId)
```

Comment:
Er det ikke bedre å tilpasser metoden isDigitalVarselEnabled?

(da vil det sørge for at andre varsler også håndteres riktig, og ikke bare dialogmøteinnkalling)

Reply:
Stemmer. fikser

Reply:
Da er  det fikset @geir-waagboe